packages/p2p-v2/jest.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
     ...baseConfigForPackages,
     moduleNameMapper: {
         '@/(.*)': '<rootDir>/src/$1',
+        '@deriv/api': '<rootDir>/../api/src',
```

Comment:
basically tells Jest if it can't find deriv/api go outside of p2p-v2 and find it in packages/api